hooks/blueprint

```diff
@@ -4,8 +4,8 @@ set -eu
 declare -a merge
 
 validate_features nfs-volume-services small-footprint \
-                  local-db mysql-db postgres-db \
-                  db-internal-postgres db-external-mysql db-external-postgres \
+                  local-db local-ha-db mysql-db postgres-db \
+                  db-internal-postgres db-ha-internal-postgres db-external-mysql db-external-postgres \
```

Comment:
`db-internal-postgres` is a legacy feature name, we don't need to mirror it with the new HA stuff.